src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,8 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
 public interface MachineProducer {
```

Comment:
**The task is not DONE**
![image](https://user-images.githubusercontent.com/20326858/207837857-7de15fc3-d4ef-4c1c-bb3b-1919ef6ffdc0.png)